backend/app/lifespan.py

```
@@ -216,18 +215,16 @@ async def _redis_subscriber():
     finally:
         try:
             pubsub.close()
-        except:
+        except Exception:
             pass
```

Ruff (0.15.7)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)